plugins/StoneTheme/themes/stone_theme.json

```diff
@@ -100,6 +100,7 @@
 			"footer": "#949494",
 			"tooltip": "#0D1321",
 			"default_button": "#818181"
+			}
```

Comment:
This is invalid JSON now:

![Bildschirmfoto 2021-03-04 um 09 32 43](https://user-images.githubusercontent.com/33347616/109934261-9a5eaa80-7ccc-11eb-8353-79979a564960.png)